src/components/BigCupom/index.jsx

```diff
@@ -139,14 +139,14 @@ const CupomDescription = styled.div`
 `;
 
 const BigCupom = ({
-  image, title, description,
+  image, title, description, textColor, titleColor, bg, circleColor,
```

Comment:
If the idea is to leave these new properties as optional, it would be interesting to add default values. "This way, you won't need to deal with fallbacks in styles either"

Reply:
Some default values was created, you can see here - c83abbd1722a4b3b85e31f5dac5395400f065431